src/webgpu/api/validation/encoding/cmds/render/draw.spec.ts

```diff
@@ -402,33 +402,40 @@ success/error as expected. Such set of buffer parameters should include cases li
   )
   .params(u =>
     u
-      .combine('drawType', ['draw', 'drawIndexed', 'drawIndirect', 'drawIndexedIndirect'] as const)
-      .combine('boundVertexBufferSizeState', ['zero', 'exile', 'enough'] as const)
-      .combine('boundInstanceBufferSizeState', ['zero', 'exile', 'enough'] as const)
-      .combine('zeroVertexStrideCount', [false, true] as const)
-      .combine('zeroInstanceStrideCount', [false, true] as const)
-      .combine('arrayStrideState', ['zero', 'exact', 'oversize'] as const)
-      .combine('attributeOffsetFactor', [0, 1, 2, 7]) // the offset of attribute will be factor * MIN(4, sizeof(vertexFormat))
+      // type of draw call
+      .combine('type', ['draw', 'drawIndexed', 'drawIndirect', 'drawIndexedIndirect'] as const)
+      // the state of vertex step mode vertex buffer bound size
+      .combine('VBSize', ['zero', 'exile', 'enough'] as const)
+      // the state of instance step mode vertex buffer bound size
+      .combine('IBSize', ['zero', 'exile', 'enough'] as const)
+      // is the vertex stride count zero
```

Comment:
~is~， same to line 413

Reply:
The parameter `VStride0` stand for `zeroVertexStrideCount`, indicating that if the vertex stride count is zero. Change the comment to "should the vertex/instance stride count be zero".